ci/README.md

```diff
@@ -0,0 +1,210 @@
+- C++ interop tests / Run (macOS-10.15, clang-10.0.0)
+- C++ interop tests / Run (macOS-10.15, clang-9.0.0)
+- C++ interop tests / Run (macOS-10.15, clang-8.0.0)
+- C++ interop tests / Run (ubuntu-18.04, clang-10.0.0)
+- C++ interop tests / Run (ubuntu-18.04, clang-9.0.0)
+- C++ interop tests / Run (ubuntu-18.04, clang-8.0.0)
+- C++ interop tests / Run (ubuntu-18.04, g++-9)
+- C++ interop tests / Run (ubuntu-18.04, g++-8)
+- C++ interop tests / Run (ubuntu-18.04, g++-7)
+- C++ interop tests / Run (ubuntu-18.04, g++-6)
+- C++ interop tests / Run (ubuntu-18.04, g++-5)
+- C++ interop tests / Run (ubuntu-20.04, clang-13.0.0)
+- C++ interop tests / Run (ubuntu-20.04, clang-12.0.0)
+- C++ interop tests / Run (ubuntu-20.04, clang-11.0.0)
+- C++ interop tests / Run (ubuntu-20.04, g++-11)
+- C++ interop tests / Run (ubuntu-20.04, g++-10)
+- C++ interop tests / Run (ubuntu-20.04, g++-9)
+
+Github action to test for C++ interoperability
+
+Most tests in the test-suite run on the CI when it comes to cross-platform testing.
+However, the dlang auto-tester uses a somewhat old host C/C++ compiler.
+This is good for testing compatibility with e.g. LTS distributions, but becomes problematic when we want to test more cutting-edge features, such as newer C++ standards (C++17, C++20, etc...).
+This is the reason why we have this action: we have full control over the toolchain, and it's cross platform.
+
+### Codecov
+
+**Config**: [.codecov.yml](https://github.com/dlang/dmd/blob/master/.codecov.yml)
+
+**Checks**:
+- codecov/patch
+
+Codecov checks code coverage, meaning that the lines of code you modified are executed at least once in the test suite.
+While there are currently no hard constraints on coverage for a Pull Request to pass the check, it does emit warnings for modified lines that are not covered into the "Files Changed" tab.
+These should be taken seriously, since untested code is likely to introduce bugs, and can easily be accidentally broken by future changes.
+
+### CirrusCI
+
+**Config**: [.cirrus.yml](https://github.com/dlang/dmd/blob/master/.cirrus.yml)
+
+**Checks**:
+- FreeBSD 11.4 x64, DMD (bootstrap)
+- FreeBSD 12.2 x64, DMD (coverage)
+- FreeBSD 12.2 x64, DMD (latest)
+- Ubuntu 18.04 x64, DMD (bootstrap)
+- Ubuntu 18.04 x64, DMD (latest)
+- Ubuntu 18.04 x64, GDC
+- Ubuntu 18.04 x64, LDC
+- Ubuntu 18.04 x86, DMD (bootstrap)
+- Ubuntu 18.04 x86, DMD (coverage)
+- Ubuntu 18.04 x86, DMD (latest)
+- macOS 10.15 x64, DMD (bootstrap)
+- macOS 11.x x64, DMD (coverage)
+- macOS 11.x x64, DMD (latest)
+- macOS 12.x x64, DMD (coverage)
+- macOS 12.x x64, DMD (latest)
+
+Cirrus tests DMD on Posix platforms.
+
+Since DMD is written in D, a "Host D Compiler" is needed to compile it.
+Various host compilers are tested, such as GDC, LDC, the latest DMD, and an older verison of DMD (bootstrap).
+Note that the GDC and LDC targets only test a different host compiler, they still compile the DMD compiler only.
```

Comment:
This phrase is kind of ambiguous. I don't understand what you are trying to say.

Reply:
Max said "Probably worth explicitly noting that the LDC/GDC tests don't test the changes on top of ldc and gdc but rather compiling dmd with them."

Reply:
Then I would rephrase to: "Note that the GDC and LDC targets do not run any tests, the simply build the latest version of the dmd frontend."

Reply:
Don't they run the test suite?